hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -587,10 +601,10 @@ void logSlowRpcCalls(String methodName, Call call,
     final double threeSigma = rpcMetrics.getProcessingMean() +
         (rpcMetrics.getProcessingStdDev() * deviation);
 
-    long processingTime =
-            details.get(Timing.PROCESSING, rpcMetrics.getMetricsTimeUnit());
+    long processingTime = details.get(Timing.PROCESSING, rpcMetrics.getMetricsTimeUnit());
```

Comment:
`rpcMetrics.getMetricsTimeUnit` is not necessary always in Millis. We need to convert getLogSlowRPCThresholdMs() into the same TimeUnit before we can do the comparison.

Reply:
Get it.